lib/ble_manager.dart

```diff
@@ -2,6 +2,13 @@ part of flutter_ble_lib;
 
 typedef RestoreStateAction = Function(List<Peripheral> restoreStateIdentifier);
 
+enum LogLevel { NONE, VERBOSE, DEBUG, INFO, WARNING, ERROR }
+
+LogLevel logLevelFromString(String logLevelName) {
```

Comment:
Should this be available to user? It looks like an internal function.
`InternalBleManager._logLevelFromString(String logLevelName)`?

Reply:
If I see correctly it's only used in `log_level_mixin.dart` so it should be there as a private function.